src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -1388,12 +1388,12 @@ public class OngoingTasksResult : IDynamicJson
         public List<OngoingTask> OngoingTasksList { get; set; }
         public int SubscriptionsCount { get; set; }
         
-        public List<PullReplicationDefinition> PullReplicationDefinitions { get; set; }
+        public List<PullReplicationDefinition> PullReplication { get; set; }
```

Comment:
Plural would be better.

Reply:
you are absolutely right. will fix that - thanks!

Reply:
done